frontend/app/work_packages/view_models/parent-relations-handler.js

```diff
@@ -28,7 +28,10 @@
 
 module.exports = function(CommonRelationsHandler, WorkPackageService, ApiHelper) {
   function ParentRelationsHandler(workPackage, parents, relationsId) {
-      var handler = new CommonRelationsHandler(workPackage, parents, relationsId);
+      var relations = parents.filter(function(parent) {
```

Comment:
I am also not sure why it is called `parents`, when there can only ever be a single parent. But I could imagine, that there are reasonsfor that... oO